tests/sentry/migrations/test_0007_update_detected_subscription_interval.py

```diff
@@ -0,0 +1,44 @@
+from sentry.testutils.cases import TestMigrations
+from sentry.uptime.models import ProjectUptimeSubscriptionMode, UptimeSubscription
+
+
+class UpdateAutoDetectedActiveIntervalSecondsTest(TestMigrations):
+    migrate_from = "0006_projectuptimesubscription_name_owner"
+    migrate_to = "0007_update_detected_subscription_interval"
+    app = "uptime"
+
+    def setup_before_migration(self, apps):
+        self.uptime_subscription = self.create_uptime_subscription(
+            url="http://sintry.io", interval_seconds=300
+        )
+        self.create_project_uptime_subscription(
+            uptime_subscription=self.uptime_subscription,
+            mode=ProjectUptimeSubscriptionMode.AUTO_DETECTED_ACTIVE,
+        )
+
+        self.uptime_subscription_unchanged = self.create_uptime_subscription(
```

Comment:
```suggestion
        # Will not change because it is in onboarding mode
        self.uptime_subscription_unchanged = self.create_uptime_subscription(
```

Reply:
I'm just going to leave as-is so I can merge, usually these migration tests are disabled a short while after merging since they break anyway